.gitmodules

```diff
@@ -70,3 +70,18 @@
 [submodule "quicklogic/designs"]
 	path = quicklogic/designs
 	url = https://github.com/QuickLogic-Corp/ql_designs.git
+[submodule "third_party/ql-eos-s3"]
+	path = third_party/ql-eos-s3
+	url = https://github.com/QuickLogic-Corp/EOS-S3
+[submodule "third_party/ql-pp3e"]
+	path = third_party/ql-pp3e
+	url = https://github.com/QuickLogic-Corp/PolarPro3E.git
+[submodule "quicklogic/common/utils/fasm-utils"]
+	path = quicklogic/common/utils/fasm-utils
+	url = https://github.com/QuickLogic-Corp/quicklogic-fasm-utils.git
```

Comment:
I wonder if this can be actually generalized and upstreamed into the fasm library?

Reply:
Indeed it could. This one is used only by `quicklogic-fasm` so probably we can integrate both into one.